src/components/checkbox/checkbox.spec.js

```diff
@@ -246,5 +247,115 @@ describe('mdCheckbox', function() {
       expect(isChecked(checkbox)).toBe(false);
       expect(checkbox.hasClass('ng-invalid')).toBe(true);
     });
+
+    describe('with the indeterminate attribute', function() {
+
+      it('should set indeterminate attr to false by default', function() {
```

Comment:
I disagree with this test; I believe it should be set to true by default to follow our existing conventions.